src/main/java/org/openstreetmap/atlas/geography/atlas/BareAtlas.java

```diff
@@ -235,6 +238,7 @@ else if (thisEntity instanceof Relation)
                     final RelationMemberList thatMembers = ((Relation) thatEntity).members();
                     if (!thisMembers.equals(thatMembers))
                     {
+                        logger.error("ERRORRRRR: {} vs {}", thisMembers, thatMembers);
```

Comment:
this is the only acceptable error format

Reply:
Heh. There's a lot of junk logging in here due to a confusing bug. That will not be in the final PR :)